okio/src/main/java/okio/RealBufferedSink.kt

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright (C) 2014 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okio
+
+import java.io.EOFException
+import java.io.IOException
+import java.io.OutputStream
+import java.nio.ByteBuffer
+import java.nio.charset.Charset
+
+internal class RealBufferedSink(
+  @Suppress("RedundantVisibilityModifier") // Required to keep JApicmp happy.
```

Comment:
need to figure out how to get it to not check package-private types

Reply:
It shouldn't be, although `internal` means `public` in bytecode. But still the old version was proper package private.

Reply:
I didn’t need `@Suppress("RedundantVisibilityModifier")` or `public`. Just `@JvmField`. That’s necessary because the field itself went from package-private to private.